xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -209,7 +204,6 @@
       "items": {
         "type": "object",
         "properties": {
-          "playercoreid": { "type": "integer", "minimum": 1, "required": true },
           "name": { "$ref": "Global.String.NotEmpty", "required": true },
           "type": { "type": "string", "enum": [ "internal", "external", "remote" ], "required": true },
```

Comment:
"pa" was for PAPlayer. I agree with you and already changed to music and remote: https://github.com/FernetMenta/xbmc/commit/4362f1bb2fbab992dde1ef14cfc14a2f6d7d2f63